.github/workflows/_unit-tests-client.yml

```diff
@@ -41,6 +41,7 @@ jobs:
         run: |
           APP_ENV=dev APP_DEBUG=0 docker-compose -f docker-compose.yml -f docker-compose.ci.client-unit-tests.yml up -d --no-deps client-unit-tests pact-mock
           sleep 3
+          docker-compose -f docker-compose.yml -f docker-compose.ci.client-unit-tests.yml exec -T client-unit-tests chown root /tmp/commonpasswords.txt
```

Comment:
this doesn't always cause a problem and couldn't work out why but for some caches the fact that unit tests are run as root and this file is owned by www-data, it causes an issue. This isn't part of the build and this 'fixes' the issue. I did try running unit tests as www-data but that caused other permission issues.